Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Status.hs

```diff
@@ -58,6 +60,9 @@ handler transporterId req = do
         DRide.NEW -> do
           bookingDetails <- SyncRide.fetchBookingDetails ride booking
           driverInfo <- QDI.findById (cast ride.driverId) >>= fromMaybeM DriverInfoNotFound
+          vehicle <- QVeh.findById ride.driverId >>= fromMaybeM (DriverWithoutVehicle ride.driverId.getId)
```

Comment:
Why vehicle is required here? If vehicle is being remove from the driver at time on on-status being called? Or it got changed in b/w?